system/jlib/jtrace.cpp

```diff
@@ -879,6 +1167,16 @@ void CTraceManager::initTracerProviderAndGlobalInternals(const IPropertyTree * t
                     exporter = otlp::OtlpGrpcExporterFactory::Create(opts);
                     DBGLOG("Exporting traces via OTLP/GRPC to: (%s)", opts.endpoint.c_str());
                 }
+                else if (stricmp(exportType.str(), "JLog")==0)
+                {
+                    StringBuffer logLevel;
+                    if (exportConfig->hasProp("@logLevel"))
```

Comment:
We are moving away from using a loglevel to control what is traced - better to use individual feature flags instead.